gym_locm/engine.py

```diff
@@ -670,6 +717,11 @@ def _act_on_draft(self, action: Action):
 
     def _act_on_battle(self, action: Action):
         """Execute the actions intended by the player in this battle turn"""
+
+        # Reset drawn cards for both players
+        for p in self.players:
+            p.drawn_cards = []
```

Comment:
@dfpetrin Hi Derrik! Asking just in case: is this correct? Did we want to empty the list of drawn cards at every battle turn?